src/components/Label/Label.module.scss

```diff
@@ -29,77 +33,115 @@
   transform-origin: bottom left;
 }
 
-.kindFill.colorPrimary {
-  background-color: var(--primary-color);
-}
+.kindFill {
+  &.colorPrimary {
+    background-color: var(--primary-color);
 
-.kindFill.colorPrimary svg path {
-  fill: var(--primary-color);
-}
+    svg path {
+      fill: var(--primary-color);
+    }
 
-.kindFill.colorDark {
-  background-color: var(--inverted-color-background);
-}
+    &.clickable:hover,
+    &.clickable:active {
+      background-color: var(--primary-hover-color);
+    }
+  }
 
-.kindFill.colorDark svg path {
-  fill: var(--inverted-color-background);
-}
+  &.colorDark {
+    background-color: var(--inverted-color-background);
```

Comment:
That's not related to your changes, but I spotted that we have a color token issue here in dark themes 
![image](https://github.com/mondaycom/monday-ui-react-core/assets/104433616/51abd053-8c1c-4e72-b799-a6086727f931)
In our storybook we have a theming switcher, which is always nice to check when adding usage of new tokens. Here is how it can be found
![image](https://github.com/mondaycom/monday-ui-react-core/assets/104433616/9e0c376f-73e7-4f73-84c7-22c3380a1bb8)

I propose we tackle it in a separate task. Could you please add a corresponding task to the board and solve it in a separate PR? Once it's merged to master, let's just update from master here and make sure it's working fine with clickable ability. Or we can do the other way around - whatever you prefer

Reply:
Fixed in this PR, changed it to the correct token 🙏🏼